.editorconfig

```diff
@@ -19,3 +19,7 @@ ij_php_concat_spaces = true
 ij_php_phpdoc_param_spaces_between_name_and_description = 1
 ij_php_phpdoc_param_spaces_between_tag_and_type = 1
 ij_php_phpdoc_param_spaces_between_type_and_name = 1
+
+[*.scss]
```

Comment:
Update some other properties here if you want to change the scss file configuration @cslant/d-ngheo-kh